docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+
+For instance, it is in-principle possible to misappropriate ``sktime`` resource, while
+being in formal compliance with specific national laws, institutional laws, and local policies.
+Irrespective of this, such an action would still count as a severe CoC violation.
+
+Examples of not acceptable behaviour
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* obtaining a resource opportunity through ``sktime``,
+  then retreating or resigning ``sktime`` roles to declare that the opportunity is now in a personal role and not on behalf of ``sktime``.
+* pretending to communicate on behalf of ``sktime``, or unauthorized use of the ``sktime`` brand,
+  including but not limited to violations of the BSD 3-clause license
+* soft refusal to implement best anti-corruption and anti-bribery practice, e.g.,
+  soft refusal to make budgets, invoives, or spending records transparent to the ``sktime`` community
+* attempts to suppress criticism of bad practice through invoking the code of conduct, e.g., tone policing or DARVO
+
+Investigating CoC violations in resource allocation decisions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CoC violations when making resource allocation decisions are rarely accompanied by violation of communicative norms.
+
+Such CoC violations can be committed by an individual, or by a networked group of individuals.
+
+DARVO, intimidation, noise generation, and plausible deniability may be common tactics found in an investigation.
+
+It is hence crucial to establish the facts.
+
+It is also crucial to investigate circumstantial evidence, as facts may be obfuscated.
+Important pieces of circumstantial evidence for a CoC violation are:
+
+* cui bono - the individual or group influencing the decision being a direct or indirect
+  beneficiary of the changed/influenced resource allocation decision
+* substantial "value" of the resource allocation decision in question, e.g., in the order of a month's living wages or above
+* premeditation, i.e., indication of substantial, careful consideration and planning to change the outcome of decision making to one's benefit,
+  or prevent/hinder decision making by the ``sktime`` community
+* removing the decision making capacity from the ``sktime`` community,
+  e.g., by creating accomplished facts or not complying with ``sktime`` community decisions.
+* obfuscation of the paper trail, e.g., soft refusal or inability to provide paper documentation such as budgets, invoices or reports.
+* hindering of the investigation itself, e.g., by not engaging with it to the fullest extent possible, creating community dissent around it, etc
+* promoting decision making on ``sktime`` governance that would weaken accountability,
+  transparency, decision making capacity, investigation practices, or adherence to best practice in resource allocation
+* despite better knowledge - evidence that the decision maker is aware of best practice and nonetheless acts against it.
```

Comment:
I think "Ignorance of the law is no excuse" should be here. If a CC member claims that they simply didn't know they couldn't transfer the OC money to their own pocket - it's not an excuse, no `evidence that the decision maker is aware of best practice` is needed.

Reply:
excellent point - I have added this as clarification. Let me know if you would suggest to improve the formulation.